### PR TITLE
[#102] [backend] messages-auto-clear

### LINKED ISSUE #102: [backend] clear chat feature

### PR DESCRIPTION
**Pull request by kirillidk**
- now for every 50 messages (user's and assistant's), only the last 25 messages will be retained, improving database storage efficiency (numbers are debatable)
- minor fix in `LlmReminderService`
- minor fix in `WordsReminderService`

Closes #102

### CODE REVIEW

File: backend/src/main/java/com/lingvoFriend/backend/Services/ChatService/ChatService.java [hunk 1]
```diff
@@ -53,6 +54,13 @@ private Message generateResponseImpl(UserModel user) {
             return languageLevelService.evaluate(user);
         }
 
+        // every 70 user's or assistant's messages we'll cut List<Messages> to 30 last
```
Comment (sharkov63) [inline backend/src/main/java/com/lingvoFriend/backend/Services/ChatService/ChatService.java:57]: The numbers in this comment outdated. I suggest to remove the comment completely

File: backend/src/main/java/com/lingvoFriend/backend/Services/UserService/UserService.java [hunk 2]
```diff
@@ -93,4 +86,23 @@ public String getCefrLevel(String username) {
         UserModel user = findOrThrow(username);
         return user.getCefrLevel();
     }
+
+    public void clearMessages(UserModel user) {
```
Comment (sharkov63) [inline backend/src/main/java/com/lingvoFriend/backend/Services/UserService/UserService.java:90]: nit `cutoffMessages`? since we don't clear it completely

Comment (kirillidk) [reply backend/src/main/java/com/lingvoFriend/backend/Services/UserService/UserService.java]: done

File: backend/src/main/java/com/lingvoFriend/backend/Services/ChatService/WordsReminderService.java [hunk 3]
```diff
@@ -7,6 +7,7 @@
 import com.lingvoFriend.backend.Services.ChatService.models.Word;
 import com.lingvoFriend.backend.Services.UserService.UserService;
 
+import com.lingvoFriend.backend.Services.UserService.UserService;
```
[the remark this answers is not in the document]
Comment (kirillidk) [reply backend/src/main/java/com/lingvoFriend/backend/Services/ChatService/WordsReminderService.java]: done

**Pull request merged**